packages/stockflux-components/src/components/popups/PopupWindow.js

```diff
@@ -26,11 +26,10 @@ const PopupWindow = ({ message, options, children }) => {
       saveWindowState: false,
       showTaskbarIcon: false,
       resizable: false,
-      backgroundColor: '#28313D',
       waitForPageLoad: true,
       alwaysOnTop: true,
-      defaultWidth: 300,
-      defaultHeight: 100
+      defaultWidth: 270,
```

Comment:
Should this be 275? The spec width looks like it's 50px + 50px + 175px = 275px

Reply:
the specs say 270x145, but if you look at the maths for the individual sections it adds up to 275. unsure which is the correct ones will confirm with UX.

Reply:
Confirmed the width should be 270.